src/_locales/zh_TW/messages.json

```diff
@@ -409,40 +409,40 @@
     "hash": "21667c5a30f89561cea02351172b9c91"
   },
   "optionUseMonospaceFont": {
-    "message": "使用等寬字體",
+    "message": "使用等寬字型",
     "description": "This is an option shown in the add-on settings.",
     "hash": "69191273c436064be8c4f5d6a8f675f5"
   },
   "optionDebugMode": {
-    "message": "啟用除錯模式",
+    "message": "啟用偵錯模式",
     "description": "This is an option shown in the add-on settings.",
     "hash": "de6506af4973e0980d64f242fba11a19"
   },
   "optionDebugModeDescr": {
-    "message": "僅用在控制台取得更詳細的日誌輸出用於匯報問題...等等。",
+    "message": "僅用來從控制臺輸出更詳細的記錄，用於回報錯誤等。",
     "description": "The description shown for the debug mode (optionDebugMode).",
     "hash": "2aff10770796694827ec4253860e0771"
   },
   "translatorCredit": {
-    "message": "該擴充元件正體中文翻譯由 $TRANSLATORS$ 參考 Cye3s的 zh_CN 完成。",
+    "message": "本附加元件正體中文翻譯由 $TRANSLATORS$ 參考了 nioTW 和 Cye3s 的翻譯來完成。",
     "description": "The credit text for the translator. See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translator-credit-inside-of-add-on for how to translate this.",
     "placeholders": {
       "translators": { "content": "$1", "example": "<a href=\"https://github.com/rugk/\">@rugk</a>" }
     },
     "hash": "a6f5deecd624d5f8cd58350c72dd8fdf"
   },
   "translatorLink": {
-    "message": "https://github.com/niotw",
+    "message": "https://github.com/Eee14",
     "description": "The link to the translator's GitHub profile.",
     "hash": "4775fb2f88fd33d02b871e5021dec3e9"
   },
   "translatorUsername": {
-    "message": "nioTW",
```

Comment:
Please keep the reference to the old translator, as you did obviously not replace all translations of them.

To honor your work, you can credit yourself here and provide a link (in `translatorLink`) and a name (in `translatorUsername`). Also, you should add yourself to the CONTRIBUTORS.md file and the changelog as a translator to [give credits here](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#credit). Also see the section on how multiple translators are to be credited.

See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translations for how these placeholders are translated, especially [the official doc](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Internationalization#Providing_localized_strings_in__locales).